Libraries/VLM/Models/Paligemma.swift

```diff
@@ -0,0 +1,727 @@
+// Copyright © 2024 Apple Inc.
+
+// port of https://github.com/Blaizzy/mlx-vlm/tree/main/mlx_vlm/models/paligemma
+
+import CoreImage
+import Foundation
+import Hub
+import MLX
+import MLXFast
+import MLXLMCommon
+import MLXNN
+import Tokenizers
+
+// MARK: - Language
```

Comment:
First cut at a port of https://github.com/Blaizzy/mlx-vlm/tree/main/mlx_vlm/models/paligemma

Reply:
Note: this builds, loads weights and "runs" but doesn't produce any output -- still needs to be debugged.

Reply:
it should be usable as an example of the structure I think we need